scripts/aura-test.sh

```diff
@@ -1,9 +1,23 @@
 #!/bin/bash
-cargo build -j $(nproc) --release --features final $CARGOFLAGS
-git clone https://github.com/paritytech/parity-import-tests
-cp target/release/parity parity-import-tests/aura/parity
-cd parity-import-tests/aura
+rm -rf parity-import-tests
+rm -rf /tmp/aura-test-data
+cargo build -j $(nproc) --release
+target/release/parity -v
 echo "Start Aura test"
-parity import blocks.rlp --chain chain.json
-parity restore snap --chain chain.json
+target/release/parity import test/parity-import-tests/aura/blocks.rlp --chain test/parity-import-tests/aura/chain.json -d /tmp/aura-test-data
```

Comment:
@General-Beck there are now multiple import tests. Can you rewrite this as a loop to be future-proof?